recipes/libyuv/all/conanfile.py

```diff
@@ -0,0 +1,49 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibyuvConan(ConanFile):
+    name = "libyuv"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://chromium.googlesource.com/libyuv/libyuv/"
+    description = "libyuv is an open source project that includes YUV scaling and conversion functionality."
+    topics = ["YUV", "libyuv", "google", "chromium"]
+    license = "BSD"
+    settings = "os", "compiler", "arch", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False]}
+    default_options = {'shared': False,
+                       'fPIC': True}
+    requires = ("libjpeg-turbo/2.0.5")
```

Comment:
```suggestion
    options = {"shared": [True, False],
               "fPIC": [True, False]
               "jpeg": [False, "libjpeg", "libjpeg-turbo"]}
    default_options = {"shared": False,
                       "fPIC": True
                       "jpeg": "libjpeg"}
```

libjpeg is optional and can be requested from libjpeg or turbo.

Reply:
fixed